pattoo/api/portal/react/src/UsersHead.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.